src/templates/status.html

```diff
@@ -26,4 +29,21 @@
                 }
             })
     }
+
+    function getQueuePosition(runId) {
+        $.ajax(
+            {
+                url: "./" + runId + "/position/",
+                success: function (result) {
+                    let queuePositionString = "Your job is scheduled after " + result + " other runs";
```

Comment:
Let's changed the "after X other runs" to "after X other _jobs_" just to make our language more consistent.